ci_lint.sh

```diff
@@ -2,4 +2,3 @@ set -ex
 
 ./manifest
 ./lint
-./diff-manifest.py
```

Comment:
Yeah, that's intentional because it doesn't really do anything. There's no way the check can fail and it doesn't expose any useful information because we effectively do the same thing with the list of changed tests that are posted to the PR.